src/server.py

```diff
@@ -13,6 +14,7 @@
 
 # The Flask app
 app = Flask(__name__)
+CORS(app)
```

Comment:
Do we need cross site scripting?

Reply:
Honestly, I use it (finally this is an API, there are 3rd part applications accessing).

Then, you can stripe it if you don't need it